APITesters/ObjCAPITester/ObjCAPITester/RCBackendErrorCodeAPI.m

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  RCBackendErrorCodeAPI.m
+//
+//  Created by Madeline Beyl on 9/7/21.
+
+#import "RCBackendErrorCodeAPI.h"
+@import RevenueCat;
+
+@implementation RCBackendErrorCodeAPI
+
++ (void)checkEnums {
+    RCBackendErrorCode errCode = RCBackendStoreProblem;
```

Comment:
kinda surprised that these are public in the first place, I think it might have been an oversight on our part

Reply:
We can remove them, but they have been public for a while, so I'm apprehensive about it. Regardless, I think that's another PR if we decide.